Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM mhalagan1nmdp/gfe-base:latest
+FROM mhalagan1nmdp/gfe-base:latest 
 
-COPY bin/* opt/
-COPY requirements.txt opt/
+ADD bin/* /opt/
```

Comment:
Why the change to `ADD` ? `ADD` is only for URLs or gzip files. See Docker Best Practices: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

Reply:
moved back to COPY where appropriate.